### ISSUE TITLE
Regarding cross corpus accuracy

### ISSUE THREAD

**Issue by VSK02101996** [ECNU-Cross-Innovation-Lab/LGCCT#1]
How it will work for some kind of reviews or interviews (real world) ?

**Comment by Trert111** [comment]
Our code is based on [`MUlT`](https://github.com/yaohungt/Multimodal-Transformer) and supports CMU-MOSEI, CMU-MOSI and IEMOCAP. The preprocessed data can be obtained from the MUlT. If you want to use custom dataset, you should modify [dataset.py](https://github.com/ECNU-Cross-Innovation-Lab/LGCCT/blob/main/src/dataset.py) and adjust the hyperparameters like hidden dim.

**Comment by VSK02101996** [comment]
Where is that .py file for hyperparameters ?

**Comment by VSK02101996** [comment]
For custom data I need to preprocess using https://github.com/yaohungt/Multimodal-Transformer this right ?

**Comment by Trert111** [comment]
> For custom data I need to preprocess using https://github.com/yaohungt/Multimodal-Transformer this right ?

The benchmark datasets like MOSEI, MOSEI, and IEMOCAP can be downloaded directly in  https://github.com/yaohungt/Multimodal-Transformer. For custom datasets, you should preprocess data following its standard practice or by yourself. This may require large modification and thus we recommend just referring to the code of our model and merging the model to your custom data pipeline. The default hyperparameter for the network can be referred in [main.py](https://github.com/ECNU-Cross-Innovation-Lab/LGCCT/blob/main/main.py).

**Comment by VSK02101996** [comment]
What is meaning of standard practice like calculating mfcc , chroma etc or something else ? Also merging means creating model for custom data and merge that model with your model ; is it right ? @Trert111

**Comment by VSK02101996** [comment]
Can you share the code how to do preprocessing of custom data ? @Trert111  . Atleast share how it is done on iemocap,mosi dataset ?

**Comment by Trert111** [comment]
For MOSEI and MOSI, you can refer to [CMU-MultimodalSDK](https://github.com/A2Zadeh/CMU-MultimodalSDK). For IEMOCAP, you can refer to other related repositories. Our data is from [here](https://www.dropbox.com/sh/hyzpgx1hp9nj37s/AAB7FhBqJOFDw2hEyvv2ZXHxa?dl=0) provided by MULT.

**Comment by VSK02101996** [comment]
I don't understand this lines can you please elaborate ; 

**For custom datasets, you should preprocess data following its standard practice or by yourself. This may require large modification and thus we recommend just referring to the code of our model and merging the model to your custom data pipeline.**

**Comment by Trert111** [comment]
Since our code now only supports dataset of MOSEI, MOSEI, and IEMOCAP, the custom dataset may require rewriting a large portion of code, including dataset, dataloader, hidden dim, and even the training&testing procedures. Therefore, you can write your own code for your dataset and refer to what you need in our code. The core model is in [modules](https://github.com/ECNU-Cross-Innovation-Lab/LGCCT/tree/main/modules).
